content/docs/guides/react.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Connect a React application to Neon
+subtitle: Set up a Neon project in seconds and connect from a React application
+enableTableOfContents: true
+updatedOn: '2024-07-02T07:55:54.402Z'
+---
+
+React by Facebook is an open-source front-end JavaScript library for building user interfaces based on components.
+
+Neon Postgres should be accessed from the server-side in React applications. Using the following React Meta-Frameworks, you would be able to easily connect to Neon on the server-side.
```

Comment:
```suggestion
Neon Postgres should be accessed from the server side in React applications. Using the following React meta-frameworks, you can easily configure a server-side connection to a Neon Postgres database.
```

Reply:
I know this is not a "framework", but it would still be nice to have it in the sidebar and create a card for it. Can we get away with adding it to the sidebar, here:

<img width="345" alt="image" src="https://github.com/neondatabase/website/assets/10074684/c4b2fc6a-0667-4373-bd07-cbf225fa4d4c">

If so, we can request a React card from Pixel Point and put it on the Frameworks landing page: https://neon.tech/docs/get-started-with-neon/frameworks